src/librustc/plugin/registry.rs

```diff
@@ -130,4 +135,22 @@ impl<'a> Registry<'a> {
     pub fn register_llvm_pass(&mut self, name: &str) {
         self.llvm_passes.push(name.to_owned());
     }
+
+
+    /// Register an attribute with an attribute type
+    ///
+    /// Registered attributes will bypass the `custom_attribute` feature gate
```

Comment:
This and the next two lines can be one paragraph. Also, all these sentences need `.`s.